README.md

```diff
@@ -218,12 +539,22 @@ A user might want to quickly find a container that most closely matches what the
 - ABI compatibility with the host
 - Image architecture
 
+> nitpick: "ABI compatibility with the host" depends on the host so the container should just
+> contain ABI info about the container, and then you'd search for the host's ABI
+
 #### Use Case: Dynamic Builds
 A user on a resource should be able to navigate to a web interface, select a set of software or features desired for a container, and then have it built for use. This use case is slightly different because we aren’t searching for existing containers, but building custom ones.  In terms of metadata, this use case would require:
 
 - Operating systems
 - Compatible package managers / software
 
+> not fully understanding this one -- wouldn't you want the full dockerfile to be provided, then
+> you just build it on demand? Or you could provide a spack.yaml here for something more
```

Comment:
I think people are getting more impatient and don't even want to wait for a container build, if that is possible. I die a little inside whenever I have to install something in spack from source :laughing:

Reply:
yeah but if the job is going to take 4 days and the build makes it faster... what's a little build up front?

Folks like Andrew want to basically JIT build container images for specific systems.  it's one way to deal with the MPI portability issue -- just build the right MPI into the container when the time comes. I don't necessarily agree with it (I would like portable binaries) but "just build it again" is a thing people think about.  It kind of breaks down with proprietary MPIs.